command/server/server.go

```diff
@@ -238,6 +249,45 @@ func isConfigFileSpecified(cmd *cobra.Command) bool {
 func runCommand(cmd *cobra.Command, _ []string) {
 	outputter := command.InitializeOutputter(cmd)
 
+	log.Println("Main process run isDaemon:", params.isDaemon)
+
+	// Launch daemons
+	if params.isDaemon {
+		// First time, err is not empty
+		_, err := strconv.Atoi(os.Getenv(daemon.EnvName))
```

Comment:
Why not use `len()` for environment variable condition judging?

Reply:
fix